src/OrchardCore/OrchardCore/Localization/NullLocalizerFactory.cs

```diff
@@ -49,10 +57,26 @@ public IEnumerable<LocalizedString> GetAllStrings(bool includeParentCultures)
 
             public IStringLocalizer WithCulture(CultureInfo culture) => Instance;
 
-            LocalizedHtmlString IHtmlLocalizer.this[string name] => new LocalizedHtmlString(name, name);
+            LocalizedHtmlString IHtmlLocalizer.this[string name] => new LocalizedHtmlString(name, name, true);
```

Comment:
Why `isResourcenotFound` is `true`?!!

Reply:
It's a NullLocalizer, there is no localization, hence I set it to true. But I looked in the code and couldn't find where it would be used for our purpose.

Reply:
This is useful for logging the missing resources, perhaps it doesn't add value here, but logically it's confusing